improver/lapse_rate.py

```diff
@@ -326,12 +326,43 @@ def _create_heightdiff_mask(self, all_orog_subsections):
 
         height_diff = np.subtract(self.all_orog_subsections, central_points)
         height_diff = np.absolute(height_diff)
-
-        height_diff_mask = np.where(height_diff >= self.max_height_diff, True,
+        height_diff_mask = np.where(height_diff >=
+                                    self.max_height_diff, True,
                                     False)
 
         return height_diff_mask
 
+    def _generate_heightdiff_mask(self, all_orog_subsections):
+        """
+        Function to create a mask for any neighbouring points where the height
```

Comment:
In a docstring, just say what the function does - it's not necessary to say that it's a function. Suggest dropping the first two words from the docstring.

Reply:
Sure - I copied the docstring from _create_heightdiff_mask, but more than happy to change it.